js/index.js

```diff
@@ -192,4 +191,50 @@ document.addEventListener('DOMContentLoaded', () => {
     event.preventDefault();
     showError(email.value);
   });
+
+  // Local-Storage
+  // declaring new variables
+  // const storage = [{
+  //   fname: '',
+  //   lname: '',
+  //   mail: '',
+  //   msg: '',
+  // }];
+
+  const firstName = document.querySelector('#firstname');
+  const lastName = document.querySelector('#lastname');
+  const comment = document.querySelector('#msg');
+
+  // Form information storing function
+  function populateStorage() {
+    localStorage.setItem('fname', firstName.value);
+    localStorage.setItem('lname', lastName.value);
+    localStorage.setItem('mail', email.value);
+    localStorage.setItem('msg', comment.value);
+  }
+  // Form completing with storage information
+  function getFromStorage() {
+    firstName.value = localStorage.getItem('fname');
+    lastName.value = localStorage.getItem('lname');
+    email.value = localStorage.getItem('mail');
+    comment.value = localStorage.getItem('msg');
+  }
```

Comment:
- [ ] Kindly store all the values in a single object, then you can set the `localStorage` using that single object. Please do the same when you want to get data from the `localStorage`. It is the same logic but instead of repeating for all input values, collate those values in a single object and store that object in the `localStorage` 👍

Reply:
@Gambit142 Hello Gambit. I wanted to make sure about what you are asking me. Before submiting i did create an object (the one you saw commented on your previous comment) with the each respectfull key to store the data, but the linter error said i wasn't using that object for anything so i commented it out. 
If I can make myself clear, is it ok if I bring back the commented object "const = storage" with it's keys, sava the values inside that object and THEN save that object in the localStorage with setItem and get it with getItem?

Reply:
@sntgopl, The `storage` variable which you commented out is an array. You can make it an object and then save that object in the localStorage.

```Javascript
const storage = {
    fname: firstName.value,
    lname: lastName.value,
    mail: lastName.value,
    msg: comment.value,
  };

localStorage.setItem('inputValue', JSON.stringify(storage));

#  when you want to fetch the data from localStorage, you will use

const retrievedObject = localStorage.getItem("inputValue");
const inputValue = JSON.parse(retrievedObject)
```